api/sql/tripSQL/initialize.sql

```diff
@@ -0,0 +1,13 @@
+DROP TABLE IF EXISTS Trips CASCADE;
+
+CREATE TABLE Trips (
+    id SERIAL PRIMARY KEY,
+    leader INT NOT NULL,
+    date DATETIME NOT NULL,
```

Comment:
this one is my bad, postgres uses TIMESTAMP for date/time data type, not DATETIME like is in the schema doc